### ISSUE TITLE
Expo init process exited with non-zero code: 2

### ISSUE THREAD

**Issue by jblevins1991** [expo/expo#2212]
I installed expo using `npm install expo-cli --global` and then ran `expo init app`. That results in it asking if I want to generate a Blank app or a Tab app.  I chose Blank app and then received the following results in the command prompt of visual studio.

`$ expo init app
? Choose a template: blank
[17:07:12] Downloading project files...
[17:07:17] Extracting project files...
Process exited with non-zero code: 2
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\source-map-resolve\node_modules\.bin\atob
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\create-react-class\node_modules\.bin\loose-envify
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\node-pre-gyp\node_modules\.bin\needle
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\node-pre-gyp\node_modules\.bin\rimraf
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\node-pre-gyp\node_modules\.bin\detect-libc
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\node-pre-gyp\node_modules\.bin\semver
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\node-pre-gyp\node_modules\.bin\rc
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\node-pre-gyp\node_modules\.bin\mkdirp
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\node-pre-gyp\node_modules\.bin\nopt
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\fsevents\node_modules\.bin\node-pre-gyp
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\babel-template\node_modules\.bin\babylon
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-sms\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-payments-stripe\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-react-native-adapter\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\react-native-gesture-handler\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\sane\node_modules\.bin\watch
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\metro-minify-uglify\node_modules\.bin\uglifyjs
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\fbjs-scripts\node_modules\.bin\semver
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\react\node_modules\.bin\loose-envify
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\fbjs\node_modules\.bin\loose-envify
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\tar\node_modules\.bin\mkdirp
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\react-native-svg\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\react-native-svg\node_modules\.bin\pegjs
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\send\node_modules\.bin\mime
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\babel-generator\node_modules\.bin\jsesc
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\babel-preset-es2015-node\node_modules\.bin\semver
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\prop-types\node_modules\.bin\loose-envify
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\cross-spawn\node_modules\.bin\which
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\babel-core\node_modules\.bin\babylon
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\babel-core\node_modules\.bin\json5
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\@expo\vector-icons\node_modules\.bin\generate-icon
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-image-loader-interface\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\@babel\traverse\node_modules\.bin\parser
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\@babel\register\node_modules\.bin\mkdirp
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\@babel\generator\node_modules\.bin\jsesc
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\@babel\template\node_modules\.bin\parser
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\@babel\core\node_modules\.bin\parser
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\@babel\core\node_modules\.bin\semver
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\@babel\core\node_modules\.bin\json5
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\win-release\node_modules\.bin\semver
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\react-native-safe-module\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\fancy-log\node_modules\.bin\color-support
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\envinfo\node_modules\.bin\which
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\jest-haste-map\node_modules\.bin\sane
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\regjsparser\node_modules\.bin\jsesc
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\metro-babylon7\node_modules\.bin\babylon
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-local-authentication\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\metro\node_modules\.bin\babylon
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\metro\node_modules\.bin\rimraf
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\metro\node_modules\.bin\image-size
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\metro\node_modules\.bin\json5
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\metro\node_modules\.bin\mkdirp
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-location\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-print\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo\node_modules\.bin\md5-file
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-camera\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-contacts\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\metro-cache\node_modules\.bin\mkdirp
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\react-native-branch\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\find-babel-config\node_modules\.bin\json5
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-ads-admob\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-analytics-segment\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\regexpu-core\node_modules\.bin\regjsparser
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\babel-register\node_modules\.bin\mkdirp
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-media-library\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\react-native\node_modules\.bin\rimraf
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\react-native\node_modules\.bin\metro
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\react-native\node_modules\.bin\semver
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\react-native\node_modules\.bin\envinfo
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\react-native\node_modules\.bin\mime
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\react-native\node_modules\.bin\mkdirp
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-asset\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\expo-gl\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\lottie-react-native\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\invariant\node_modules\.bin\loose-envify
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\temp\node_modules\.bin\rimraf
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\babel-traverse\node_modules\.bin\babylon
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\babylon
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\jsesc
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\loose-envify
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\needle
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\atob
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\rimraf
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\detect-libc
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\react-native
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\uglifyjs
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\md5-file
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\expo
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\parser
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\metro
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\pegjs
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\semver
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\regjsparser
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\envinfo
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\image-size
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\json5
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\watch
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\mime
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\generate-icon
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\uuid
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\rc
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\color-support
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\sane
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\which
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\mkdirp
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\node-pre-gyp
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\.bin\nopt
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\normalize-package-data\node_modules\.bin\semver
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\node-notifier\node_modules\.bin\semver
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\node-notifier\node_modules\.bin\which
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\xcode\node_modules\.bin\pegjs
ERROR: Can not create symbolic link : A required privilege is not held by the client. : C:\Users\jblevins\Documents\Code\Personal App\app\node_modules\xcode\node_modules\.bin\uuid

Process exited with non-zero code: 2
[17:08:46] Customizing project...

Your project is ready at C:\Users\jblevins\Documents\Code\Personal App\app
To get started, you can type:

  cd app
  expo start`

While it shows that I can start the project I feel as if a lot of things will end up being an issue in the future and would rather deal with it now than later.

**Comment by jblevins1991** [comment]
While I did not figure out why all of these errors occurred, it doesn't affect app development so I guess I should close this.

**Comment by emmanuel-2u** [comment]
@jblevins1991 I also had this problem when I was creating a new project, but I didn't wait for the project to be setup completely, so I wasn't able to run the project. I think the problem is cause one hasn't taken full control of the system. I fixed the error by following the steps in this video (https://www.youtube.com/watch?v=n_mmsnZ9BVE). I just created a new project and everything worked normally. Hope it helps.
